extensions/ui/org.eclipse.smarthome.ui.paper/web-src/css/components.css

```diff
@@ -611,7 +611,10 @@ md-progress-circular.md-default-theme .md-inner .md-left .md-half-circle
 }
 
 .parameter>div {
```

Comment:
This is what I have.
<img width="345" alt="screen shot 2017-05-11 at 11 51 40" src="https://cloud.githubusercontent.com/assets/5161937/25943560/93a7a484-3640-11e7-94ff-6fd9e8ced296.png">